cmd/opentelemetry/app/internal/esclient/client.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/es/config"
 )
 
+var errMissingURL = fmt.Errorf("missing Elasticsearch URL")
```

Comment:
nit: `errors.New()` is a more idiomatic approach for completely static error messages